files/anitya_cron.py

```diff
@@ -1,19 +1,29 @@
 #!/usr/bin/env python
 #-*- coding: utf-8 -*-
 
+import sys
+import logging
+import itertools
+import multiprocessing
+
 import anitya
 import anitya.app
 import anitya.lib.exceptions
 import anitya.lib.model
 
-import logging
-import sys
 
-PBAR = True
-try:
-    from progressbar import Bar, ETA, Percentage, ProgressBar, RotatingMarker
-except ImportError:
-    PBAR = False
+def update_project(project):
+    """ Check for updates on the specified project. """
+    print(project.name)
```

Comment:
You always run as python3?  or is there a ``from __future__ import print_function`` here that I'm missing.